provisioning/resources/init/snowplow_stream_enrich

```diff
@@ -10,13 +10,13 @@
 ### END INIT INFO
 
 dir="/home/ubuntu/snowplow/bin/"
-cmd="./snowplow-stream-enrich-0.10.0 --config /home/ubuntu/snowplow/configs/snowplow-stream-enrich.hocon --resolver file:/home/ubuntu/snowplow/configs/iglu-resolver.json --enrichments file:/home/ubuntu/snowplow/configs/enrichments"
+cmd="java -jar snowplow-stream-enrich-0.11.0.jar --config /home/ubuntu/snowplow/configs/snowplow-stream-enrich.hocon --resolver file:/home/ubuntu/snowplow/configs/iglu-resolver.json --enrichments file:/home/ubuntu/snowplow/configs/enrichments"
```

Comment:
it's 0.11.1

Reply:
Since nsq version of stream enrich will be used in the v0.4.0, I think there is no need to change stream enrich to 0.11.1 right now.

Reply:
We might as well use the latest components available